src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/appsettings.json

```diff
@@ -1,19 +1,19 @@
 {
     "AppSettings": {
-        "AuthenticationService": "http://localhost:5198/oauth/token",
+        "AuthenticationService": "http://localhost:8045/realms/edfi/protocol/openid-connect/token",
```

Comment:
When I first tested and the old value was still on line 3, I got an infinite loop 😆: the DMS was constantly forwarding to itself!

Reply:
I ran into the same thing! I keep treating appsettings.json like a local  .env so I assumed everyone else had a "" as a default value and only kept the change locally,